build/templates/backend/scripts/polygon_heimdall.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+{{define "main" -}}
+
+set -e
+
+INSTALL_DIR={{.Env.BackendInstallPath}}/{{.Coin.Alias}}
+DATA_DIR={{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend
+
+HEIMDALL_BIN=$INSTALL_DIR/heimdalld
+HOME_DIR=$DATA_DIR/heimdalld
+CONFIG_DIR=$HOME_DIR/config
+
+if [[ ! -d "$CONFIG_DIR" ]]; then
+  # init chain
+  $HEIMDALL_BIN init --home $HOME_DIR
+
+  # overwrite genesis file
+  cp $INSTALL_DIR/genesis.json $CONFIG_DIR/genesis.json
+fi
+
+# --bor_rpc_url: backend-polygon-bor ports.backend_http
+# --eth_rpc_url: backend-ethereum ports.backend_http
+$HEIMDALL_BIN start \
+  --home $HOME_DIR \
+  --rpc.laddr tcp://127.0.0.1:{{.Ports.BackendRPC}} \
+  --p2p.laddr tcp://0.0.0.0:{{.Ports.BackendP2P}} \
+  --laddr tcp://127.0.0.1:{{.Ports.BackendHttp}} \
+  --node tcp://127.0.0.1:{{.Ports.BackendRPC}} \
+  --bor_rpc_url http://127.0.0.1:8170 \
+  --eth_rpc_url http://127.0.0.1:8136 \
```

Comment:
What is `--eth_rpc_url`? Is it using ethereum backend?

Reply:
that's correct

Reply:
added the comment above to try and clarify:
```
# --bor_rpc_url: backend-polygon-bor ports.backend_http
# --eth_rpc_url: backend-ethereum ports.backend_http
```